packages/components/forms/src/input-group/InputGroup.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: 'InputGroup'
+type: 'component'
+status: 'stable'
+slug: /components/input-group/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/forms/src/input-group'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/form-elements-inputgroup--basic'
+typescript: ./InputGroup.tsx
+---
+
+`InputGroup` enables lets user set a button at the start or end of a input, being able to have a spacing or be collapsed.
```

Comment:
```suggestion
`InputGroup` enables the user to set a button at the start or end of an input, with some spacing or collapsed.
```

Reply:
I'm suggesting just some typo fixes and I also rephrased the second sentence a bit